test/UnitTests/Microsoft.Testing.Platform.UnitTests/Program.cs

```diff
@@ -19,8 +19,10 @@
 #if ENABLE_CODECOVERAGE
 builder.AddCodeCoverageProvider();
 #endif
-builder.AddCrashDumpProvider();
+builder.AddAppInsightsTelemetryProvider();
```

Comment:
@Evangelink Any reason telemetry wasn't enabled previously? Is it intentional for our tests to opt-out for telemetry?